src/components/list/list.js

```diff
@@ -86,7 +90,7 @@ function mdListItemDirective($mdAria, $mdConstant, $mdUtil, $timeout) {
         }
         if (hasProxiedElement) {
           wrapIn('div');
-        } else if (!tEl[0].querySelector('md-button')) {
+        } else if (!tEl[0].querySelector('md-button:not(.md-secondary):not(.md-exclude)')) {
```

Comment:
Where is `.md-exclude` documented?   At least let's provide some information in the API section.